app/Jobs/EsDocument.php

```diff
@@ -66,5 +68,18 @@ public function handle(): void
         $keyName = $model->getKeyName();
         $model->setAttribute($keyName, $id);
         $model->esDeleteDocument();
+        static::incrementStat('delete', get_class($model));
+    }
+
+    private static function incrementStat(string $action, string $class)
+    {
+        Datadog::increment(
+            config('datadog-helper.prefix_web').'.es_document',
+            1,
+            [
+                'action' => $action,
+                'class' => $class,
```

Comment:
doesn't datadog tag have specific format requirement? (snake case etc)

Reply:
it will and the class names will look weird I think? Something like `app_models_artisttrack`